### PR TITLE
Update dependencies and testing instructions

### PR DESCRIPTION
**Pull request by fred13kim**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,9 +21,15 @@ this is `sudo dnf install libudev-devel`).
...
 
 ```bash
+# Make sure to update & initialize submodules
+git submodule update --init --recursive
+
 # Set up and compile
 scons
 
+# To build fw
+scons fw
```
Comment (gary-kim) [inline README.md:28]: Could you replace the `scons` line above with `scons fw` instead?

Comment (fred13kim) [reply README.md:28]: done.

**Comment by gary-kim** [comment]
To leave as a note for the future, this is required because `eoltest`
wants to flash the Device Under Test (DUT) board itself. It does this by
using `esptool.py`. In the future, we should use the `esptool.py` that
already comes from platformio but this is a temporary fix until that is
implemented.

**Review by gary-kim — APPROVED** [review]
LGTM

**Pull request merged**